README.md

```diff
@@ -2,7 +2,13 @@
 Janitor service to cleanup resources created by [Cloud Resource Library (CRL)](https://github.com/DataBiosphere/terra-cloud-resource-lib) 
 
 ## Overview
-TODO
+![](https://app.lucidchart.com/publicSegments/view/057ef338-e869-4fb3-acd9-6ad2d7d560ba/image.jpeg)
```

Comment:
I feel CRL might not be the own client for using Janitor. Janitor can be used by anyone as long as they have the janitor-client-sa?

Reply:
Good point, done.